templates/init.html

```diff
@@ -167,7 +167,7 @@
       <span id="int"></span>
     </div>
     <div id="data">
-      <p style="text-align: center;"><i>website created by TheTechRobo</i></p>
+      <p style="text-align: center;"><i>website created by <a href="https://github.com/TheTechRobo">TheTechRobo</a></i></p>
```

Comment:
This should be https://thetechrobo.ca ideally, my GitHub profile page is currently shit

Reply:
You can edit this. I currently do not have time right now.